index.html

```diff
@@ -0,0 +1,154 @@
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, intial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+		<link href="style.css" rel="stylesheet" type="text/CSS">
+		<title >Robin Hodge's Portfolio Website</title>
+	</head>
+	<body id="top">
+		<div class="contact">
+			<div class="contact-outer-container contact-top">
+				<h2>Contact</h2>
+				<p> The best way to get in touch with me is via email, but if you prefer social media I have included these too.</p>
+				<div class="contact-container">
+					<a href="mailto:mrrobinhodge@gmail.com"><img src="email.png" alt="An email thumbnail"></a>
+					<a href="https://www.linkedin.com/in/robin-hodge/" target="_blank"><img src="linkedin.png" alt="A linkedin thumbnail"></a>
+					<a href="https://github.com/horsenelly" target="_blank"><img src="github.png" alt="A github thumbnail"></a>
+				</div>
+			</div>
+		</div>
+		</div>
+		<nav id="nav">
+			<div class="nav-left-container">
```

Comment:
this div is not closed

Reply:
also, you don't need both the nav and div here, you can actually just have nav and apply all the styling from .nav-left-container to the nav tag instead and it would do the same thing with just 1 element instead of 2

Reply:
done